airbyte-integrations/connectors/destination-elasticsearch/src/main/resources/spec.json

```diff
@@ -23,6 +23,11 @@
         "description": "If a primary key identifier is defined in the source, an upsert will be performed using the primary key value as the elasticsearch doc id. Does not support composite primary keys.",
         "default": true
       },
+      "certAsBytes": {
```

Comment:
For consistency, let's keep the name of this field same as in other connectors: `ca_certificate` (That's what we call this field in Postgres source, Postgres destination, and MySQL source connectors)

Reply:
updated, thanks